### ISSUE TITLE
Is it possible to use with multiple translation hooks in one component?

### ISSUE THREAD

**Issue by Demianeen** [nabekou29/js-i18n.nvim#45]
Just found a plugin today, it is great! This was main thing I was lacking from webstorm and now it is here and even better)

Plugin worked out of the box for me. The only issue I found is that plugin is confused when I use multiple `t` in one file. Take a look at this component for example:

```tsx
export const ContactInfoStep = memo(function ContactInfoStep({
	onNextStep,
}: RegistrationStepProps) {
	const t = useTranslations('login') // < -- one `t` for all login labels
	const tForm = useTranslations('form') // <-- and another `t` to reuse translations for fields from zod translation namespace
	...

	return (
		<>
			<AuthHeader title={t('contactInfo.title')} />
			<Stack>
				<TextField
					label={tForm('firstName')}
					...
				/>
				...
				<Button>{t('continue')}</Button>
			</Stack>
		</>
	)
})
```

As I said in the comment above, I use two different `t` to reuse already translated field names from zod translation namespace while having separate namespace for everything else.

In this case, plugin becomes confused and only creates preview text for first `t`, while on edit command takes namespace from second `t`

https://github.com/user-attachments/assets/2a534769-c4f7-44d3-a057-41afc71dc638

On video I try to add translation to a login namespace, but it adds it to form namespace. I also showed in the end how preview partially works.

Sorry for quality, GitHub has 10mb upload limit. You can check video in full quality [here](https://share.cleanshot.com/fmhqQdT3)

**Comment by nabekou29** [comment]
@Demianeen 
Thank you for your feedback and for using the plugin!

Fixing this issue will require fundamental changes to the LSP, so it might take some time. However, I consider this a high-priority problem and will work on addressing it as soon as possible.

**Comment by nabekou29** [comment]
@Demianeen 
The issue has just been fixed, and I’ve released it as [v0.5.1](https://github.com/nabekou29/js-i18n.nvim/releases/tag/v0.5.1).

If you notice anything else, please don’t hesitate to report them.


### Image

<image width="480px" src="https://github.com/user-attachments/assets/5dbc32b2-1abe-4336-ac36-66745219c6d4">

**Comment by Demianeen** [comment]
@nabekou29 
Just checked, it is working! Thank you man!
